monkey/common/agent_configuration/validators/ip_ranges.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+from marshmallow import ValidationError
+
+ip_regex = r"((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)"
+cird_notation_regex = r"([0-9]|1[0-9]|2[0-9]|3[0-2])"
```

Comment:
```suggestion
cird_notation_regex = r"(([0-2]*[0-9])|(3[0-2]))"
```

Reply:
Your suggestion matches `22222222222`, which is not a valid cidr

Reply:
s/*/?

```suggestion
cird_notation_regex = r"(([0-2]?[0-9])|(3[0-2]))"
```

But it doesn't matter if we can use `IPv4Network` instead.